README.md

```diff
@@ -609,6 +609,35 @@ class SuperArray extends Array {
 ```
 **[⬆ back to top](#table-of-contents)**
 
+### Functions are often cleaner than subclasses ###
```

Comment:
I would instead recommend a rewrite of the SuperArray example. Will do another PR with rewitten section.